doc/contributions/introduction.md

```diff
@@ -0,0 +1,80 @@
+# Introduction
+
+## Git and Github
+
+We recommend you to get familiar with the
+[git](https://guides.github.com/introduction/git-handbook) and
+[Github](https://guides.github.com) workflows before posting any changes.
+
+### Set up in a nut shell
+
+The following steps describe the process in a nut shell to provide you a basic
+template:
+
+* Create an account on [GitHub](https://github.com)
+* Fork the [shadow-utils repository](https://github.com/shadow-maint/shadow)
+* Clone the SSSD repository
+
+```
+git clone git@github.com:shadow-maint/shadow.git
```

Comment:
Should this be git clone https://github.com/shadow-maint/shadow.git ?

Reply:
Fixed